.github/CODEOWNERS

```diff
@@ -344,41 +344,38 @@
 /core/domain/prod_validation_jobs_one_off*.py @ankita240796
 /core/domain/storage_model_audit_jobs_test.py @ankita240796
 /core/jobs*.py @seanlip
-/core/platform/ @DubeySandeep
+/core/platform/ @seanlip
 /core/templates/dev/head/App*.ts @ankita240796
 /core/templates/dev/head/app.constants.ts @ankita240796
 /core/templates/dev/head/app.constants.ajs.ts @ankita240796
 /core/templates/dev/head/combined-tests.spec.ts @ankita240796
 /core/templates/dev/head/pages/interaction-specs.constants.ajs.ts @vojtechjelinek @Jamesjay4199
-/core/templates/dev/head/I18nFooter.ts @DubeySandeep
-/core/templates/dev/head/i18n-footer.directive.html @DubeySandeep
+/core/templates/dev/head/I18nFooter.ts @nithusha21
+/core/templates/dev/head/i18n-footer.directive.html @nithusha21
 /core/templates/dev/head/services/contextual/document-attribute-customization.service.ts @YashJipkate
 /core/templates/dev/head/services/contextual/meta-tag-customization.service.ts @YashJipkate
-/core/templates/dev/head/services/translation-file-hash-loader.service.ts @DubeySandeep
-/release_constants.py @ankita240796 @nithusha21 @DubeySandeep @seanlip
+/core/templates/dev/head/services/translation-file-hash-loader.service.ts @nithusha21
+/release_constants.py @ankita240796 @nithusha21 @seanlip
 
 
 # Miscellaneous.
-/__init__.py @DubeySandeep
-/core/__init__.py @DubeySandeep
-/core/controllers/__init__.py @DubeySandeep
-/core/domain/__init__.py @DubeySandeep
-/core/domain/config*.py @DubeySandeep
-/core/domain/change_domain.py @DubeySandeep
-/core/templates/dev/head/tests/ @DubeySandeep
-/core/templates/dev/head/domain/utilities/ @DubeySandeep
-/schema_utils*.py @DubeySandeep
+__init__.py @nithusha21
```

Comment:
Probably put * before this?

Reply:
I think `__init__.py` will make sure that any file named __init__.py (in any path) to be assigned to @nithusha21.

I'll add the reference link soon!

Reply:
Source: https://git-scm.com/docs/gitignore#_examples

Note: CODEOWNERS follow the same patterns as gitignore

Reply:
Understood, but I think it would be clearer to read with a "*", otherwise it's easy to misinterpret/misread this as just `/__init__.py`. What do you think?

Reply:
I have undone the changes as I found that there are lots of `__init__.py` files in different dir so I think it's better to make it explicit as it was earlier.